### ISSUE TITLE
Accessibility: Unable to add a text overlay for Story post with VoiceOver

### ISSUE THREAD

**Issue by reginabally** [wordpress-mobile/WordPress-iOS#16063]
It was reported in 3803957-zd-woothemes that a user without eyesight wasn't able to access the "A" button to add a text overlay when creating a Story post with the app.

### Expected behavior

The user would expect to be able to "tap" on the "A" button to add a text overlay with VoiceOver.

### Actual behavior

The "A" button wasn't accessible with VoiceOver.

##### Reported on WPiOS 16.8

I'm asking for the device model and iOS version and will update it here.
